contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/Order.java

```diff
@@ -34,19 +34,23 @@ public class Order {
     @ManyToOne
     @JoinColumn(name = "customer")
     @CollectionFilterParam
+    @org.springframework.data.rest.core.annotation.RestResource(rel = "d:customer")
```

Comment:
Why fully qualified instead of importing?

Reply:
Well, because there is a second one also named `RestResource` (from spring-content) that is imported.